### ISSUE TITLE
[discussions]Why do each of you maintain a Solid.js version of shadcn/ui?

### ISSUE THREAD

**Issue by LiMao00** [sek-consulting/solid-ui#91]
Why do each of you maintain a Solid.js version of shadcn/ui?
https://www.solid-ui.com/
https://github.com/sek-consulting/solid-ui

and
https://shadcn-solid.com/
https://github.com/hngngn/shadcn-solid

**Comment by sek-consulting** [comment]
Hi @LiMao00,

thanks for asking. 

When I started solid-ui as a small personal project I haven't found any other solid port. That's why I decided to make it public.
But around the same time hngngn seemd to also have started the project but I never reached out to them since it felt like the github repo was kinda on hold.

Now it looks like he picked it back up and also reserverd the domain :)

But I think there is still a major difference between both projects: 

- shadcn-solid seems to try to be a feature complete solid port of shadcn/ui. 
- solid-ui tries to be the most complete component library by porting shadcn/ui, tremor and custom components over to solid by using both kobalte & corvu to it's fullest
